packages/aws-cdk/lib/init-templates/sample-app/python/tests/unit/test_%name%_stack.template.py

```diff
@@ -1,11 +1,19 @@
-import unittest
+import json
+import pytest
 
 from aws_cdk import core
+from %name%.%name%_stack import %name.PascalCased%Stack
 
-from hello.hello_construct import HelloConstruct
 
-class Test%name.PascalCased%Stack(unittest.TestCase):
+def get_template():
+    app = core.App()
+    %name.PascalCased%Stack(app, "%name%")
+    return json.dumps(app.synth().get_stack("%name%").template)
 
-    def setUp(self):
-        self.app = core.App()
-        self.stack = core.Stack(self.app, "TestStack")
\ No newline at end of file
+
+def test_sqs_queue_created():
+    assert("AWS::SQS::Queue" in get_template())
```

Comment:
Those assertions are a little underwhelming

Reply:
agreed, let me see if I can verify the full template